docs/partials/client-cert-alt-port.md

```diff
@@ -0,0 +1,7 @@
+### Route doesn't ask for certificate
```

Comment:
Since this is fixed in the next release, maybe remove this warning and don't backport the guide?

Reply:
resolved in 0f72ec24997ff2a857897f642002ed356d336223, though we will also need to update the Troubleshooting page for the same thing.